packages/nx/src/config/misc-interfaces.ts

```diff
@@ -86,7 +86,7 @@ export interface ExecutorsJson {
 export interface ExecutorConfig {
   schema: any;
   hasherFactory?: () => CustomHasher;
-  implementationFactory: () => Executor;
+  implementationFactory: () => Promise<Executor>;
```

Comment:
I couldn't shim this one like the other two because it returns a promise or an async iterable